xml/utilities.xml

```diff
@@ -479,6 +479,32 @@ Average:    92.08    211.70 1097.30     0.26 12010.28      0.00      0.00      0
       page swapped out is being reused) or 0 (no pages have been scanned).
       The value should not drop below 30.
      </para>
+     <note>
+      <title>Understanding and addressing %vmeff values exceeding 100% in modern Linux
```

Comment:
To make it easier to run spellchecks on the file (and to make it clear what the parameter names are), I would wrap the parameter names (like '%vmeff', 'pgsteal', 'pgsteal' etc.) in this note in markup (they will then automatically be excluded from the spellcheck). I would have suggested using `<literal>` for this, but as the example above this note uses `<guimenu>` for the names, maybe continue to use `<guimenu>` here.